src/components/executive-education-2u/styles/_ExecutiveEducation2UPage.scss

```diff
@@ -0,0 +1,34 @@
+.course-summary {
+  .registration-outline {
```

Comment:
These styles don't appear to be applying. I would also recommend instead using `.bg-light-500`, `.p-3`, and `.rounded-lg` on the `div` where this class name is applied instead of defining custom styles.